### DISCUSSION TITLE
WARNING Failed to check if we can skip restoring outputs for components#build: test: GrpcFailure(DeadlineExceeded). Proceeding to check cache

### DISCUSSION THREAD

**Discussion by MyPrototypeWhat** [vercel/turborepo#6750] [Help] [answered]
<img width="1016" alt="image" src="https://github.com/vercel/turbo/assets/43230886/4973cd4e-5969-413e-8942-98e2dedc1c4c">
A cache folder was specified, but the folder is empty，I want to know how to solve this

**Comment by mehulkar** [comment, marked as answer]
This is likely not an issue with your usage of the `--cache-dir` flag. This message appears when Turborepo fails to communciate with the daemon. Specifically here, Turborepo is trying to determine whether it can skip cache restoration altogether. Let's say your task builds a `dist/` directory. Turborepo's daemon watches that `dist/` directory for file changes. If the next time you run that task, that `dist/` directory hasn't changed, Turborepo can not only skip _running_ your task, it can skip _restoring_ the outputs. This error message failed to communicate with the daemon to find this information. 

In any case, this does not have anything to do with `--cache-dir`. Hope this helps.

**Reply by MyPrototypeWhat** [reply]
I understand that this issue is not related to --cache-dir,but this folder is empty,Is it caused by "Turborepo fails to communciate with the daemon"

**Reply by mehulkar** [reply]
It should not be related to the daemon issue. 

If you're saying that the flag isn't working, please open an issue with a reproduction!
